user/tlu/hardware/src/AidaTluController.cc

```diff
@@ -607,7 +607,7 @@ namespace tlu {
   /// Writes to IPBus register to instruct the TLU to start/stop a run.
   //  state= 1 when running, 0 when stopped.
   //  On start, the internal registers are reset and the TLU issues a T0 pulse to DUTs.
-    SetWRegister("Shutter.RunActive", state);
+    SetWRegister("Shutter.RunActiveRW", state);
```

Comment:
The current  registers xml version 10 using the name _Shutter.RunActive_ , not sure if we should also correct version 10 xml.

Reply:
This seems to be a change from the latest FW only, at least according to the register information and errors printed when initializing.

Reply:
@simonspa is right - took us a few minutes to figure it out.